### ISSUE TITLE
Update the Wikidata item using Crossref / ORCiD information

### ISSUE THREAD

**Issue by GerardMeijssen** [WDscholia/scholia#1567]
This paper https://essd.copernicus.org/articles/12/61/2020/ has plenty of ORCiD references, the current Wikidata item Q100326870 has only one manually added author.. It would be cool if the author name strings can be removed and replaced with authors based on the ORCiD identifiers.

**Comment by fnielsen** [comment]
Yes. citation-js entry that is linked in Scholia can do that, but at the moment we cannot edit already entered data in Wikidata. I wonder if that is a functionality already in citation-js or it is something we can add as an issue over there. @larsgw would know

**Comment by egonw** [comment]
We should see if the Orcidator can be reinstated now...
